manage.py

```diff
@@ -46,10 +53,24 @@ def test(all=False):
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
 
+    q = Queue('default', connection=conn)
     while True:
         logging.info("Starting scrape...")
-        scrape_and_categorize_articles()
-        logging.info("Finished scrape. Let's run it back...")
+        job = q.enqueue(scrape_and_categorize_articles)
```

Comment:
You can’t categorize articles that aren’t there, and rq *does* attempt to fork tasks and complete them simultaneously (so it’s not strictly a queue, unless you add the option ```depends_on``` to the enqueue() call).